### PR TITLE
Create linux-vmss.json

### LINKED ISSUE #4: Enable e2e tests for Azure
We should set up an e2e tests for this repo

- [x] Setup e2e tests infra on Azure
- [x] Enable e2e tests for PRs
- [x] Switch to test-infra for presubmit e2e tests (presubmit tests are running for pull requests in cloud-provider-azure repo) @ritazh 
  - [x] Add e2e steps to setup test-infra via Makefile
  - [x] Update presubmit tests to the new way
  - [x] Remove current perl scripts
- [ ] Enable full periodic e2e tests for stable releases (periodic tests are running in backgroud for each release braches)
  - [ ] Add more test scenarios, e.g. conformance, correctness, alpha-features, autoscaling, scalability, multi-zone, slow and serial
  - [ ] Enable e2e for more releases, e.g. v1.13 and v1.12
  - [ ] Setup prow jobs for new test scenarios
- [ ] Add Azure features e2e testings #7
  - [ ] LoadBalancer service tests with various annotations
  - [ ] ACR image pulling tests without docker secrets setting explicitly
  - [ ] Multi availability zones tests
  - [ ] Cloud provider configuration options tests (e.g. Standard LoadBalancer and VMSS)
  - [ ] Upgrading tests (e.g. cloud-provider version updates)
- [ ] Documentation for e2e @ritazh

### PR DESCRIPTION
**Pull request by Abhik1998**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
kind feature
> /kind flake

**What this PR does / why we need it**:
PR is for Virtual Machine State set test cases.
**Which issue(s) this PR fixes**:

Refer #4 

**Special notes for your reviewer**:
New file can be tested

**Release note**:
```
action-required= test
```

### CODE REVIEW

File: tests/k8s-azure/manifest/linux-vmss.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
```
Comment (feiskyer) [inline tests/k8s-azure/manifest/linux-vmss.json]: Could you copy this from linux.json and only change `"availabilityProfile": "VirtualMachineScaleSets"` here

Comment (Abhik1998) [reply tests/k8s-azure/manifest/linux-vmss.json]: I did the same Sir

Comment (feiskyer) [reply tests/k8s-azure/manifest/linux-vmss.json]: still not same, could you do it again and make a force push?

Comment (Abhik1998) [reply tests/k8s-azure/manifest/linux-vmss.json]: Pushed

File: tests/k8s-azure/manifest/linux-vmss.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "count": 1,
+            "dnsPrefix": "{dnsPrefix}",
+            "vmSize": "Standard_F2"
+        },
+        "agentPoolProfiles": [
+            {
+                "name": "agentpool1",
+                "count": 2,
+                "vmSize": "Standard_F2",
+                "availabilityProfile": "VirtualMachineStateSets",
```
Comment (feiskyer) [inline tests/k8s-azure/manifest/linux-vmss.json]: VirtualMachineStateSets  should be `VirtualMachineScaleSets`

**Pull request merged**